geode-serialization/src/test/java/org/apache/geode/internal/serialization/filter/GlobalSerialFilterConfigurationTest.java

```diff
@@ -29,38 +29,60 @@
 
   private SerializableObjectConfig config;
   private GlobalSerialFilter globalSerialFilter;
-  private Consumer<String> logger;
+  private Consumer<String> infoLogger;
+  private Consumer<String> warnLogger;
+  private Consumer<String> errorLogger;
 
   @Before
   public void setUp() {
     config = mock(SerializableObjectConfig.class);
     globalSerialFilter = mock(GlobalSerialFilter.class);
-    logger = uncheckedCast(mock(Consumer.class));
+    infoLogger = uncheckedCast(mock(Consumer.class));
+    warnLogger = uncheckedCast(mock(Consumer.class));
+    errorLogger = uncheckedCast(mock(Consumer.class));
   }
 
   @Test
-  public void configureLogs_whenUnsupportedOperationExceptionIsThrown_withCause() {
+  public void configureLogsInfo_whenOperationIsSuccessful() {
```

Comment:
I think we should remove `configure` from the beginning of these test names since it's a `FunctionalInterface` with just one method.

Reply:
Fixed